packages/fxa-settings/src/lib/context/context-validation.ts

```diff
@@ -0,0 +1,187 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// TODO: Figure out how to port Vat. Here's a simplistic implementation for POC.
```

Comment:
I think the remaining bits here can be fixed in a follow up.

Reply:
Issue filed: FXA-6800